docs/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Home
+---
+
+![Comic: Distributed Pair
+Programmers](images/comics/dpp_0.png){:class="center"}
+
+
+## What others think 
```

Comment:
Preferred order:

* Brief explanation of Saros
* "Get Saros"
* "Good Starting Points"
* "What others think"